src/auth/account.repository.ts

```diff
@@ -20,6 +20,18 @@ import { Misc } from '../shared/misc';
 
 @EntityRepository(Account)
 export class AccountRepository extends Repository<Account> {
+  private accountInformationRepo = getRepository(
+    AccountInformation,
+    'authConnection',
+  );
+
+  private accountPasswordRepo = getRepository(
```

Comment:
Not a good way, but good for now

Reply:
what would be the best way to do this?, in the documentation I did not find anything about it

Reply:
@ernesto-glz we have connection here https://github.com/azerothcore/acore-api/blob/master/src/auth/auth.module.ts#L11

Strange that gives an error.
You have to make a repository file

Reply:
@IntelligentQuantum  It only gives an error when I am trying to use some method directly from an entity like here https://github.com/azerothcore/acore-api/blob/master/src/auth/account.repository.ts#L36